### PR TITLE
basic sattrack loader

### PR DESCRIPTION
**Pull request by d70-t**
Rough implementation for loading sattelite tracks from https://github.com/orcestra-campaign/sattracks

### CODE REVIEW

File: orcestra/sat.py [hunk 1]
```diff
@@ -112,3 +113,43 @@ def get_track_for_day(self, day):
             df["time"], format="%d %b %Y %H:%M:%S.000", exact=False
         )
         return df.set_index("time").sort_index().to_xarray()
+
+
+class SattrackLoader:
+    def __init__(self, satelite_name, forecast_day):
+        self.satelite_name = satelite_name
```
Comment (lkluft) [inline orcestra/sat.py:120]: It took me some digging in other repos to find out which satellite name is actually supported... I would at least add a doc string.

File: orcestra/sat.py [hunk 2]
```diff
@@ -112,3 +113,43 @@ def get_track_for_day(self, day):
...
+        df = df.rename(
+            {
+                "GMT": "time",
+                "SUBLAT[W+]": "lat",
+                "SUBLON[N+]": "lon",
+                "HEADING": "heading",
+            },
+            axis=1,
+        )
+        print(df)
```
Comment (lkluft) [inline orcestra/sat.py:153]: I would remove this `print` statement or at least make it a proper logging for debug purposes 😉

File: orcestra/sat.py [hunk 3]
```diff
@@ -112,3 +113,43 @@ def get_track_for_day(self, day):
...
+        df["lat"] = -df["lat"]  # east positive
```
Comment (lkluft) [inline orcestra/sat.py:154]: Wouldn't "east positive" require to change the sign of the **lon**gitude?

**Review by lkluft — CHANGES_REQUESTED** [review]
Thanks for this cool function! I proposed some minor changes to make the function a bit more user-friendly

File: orcestra/sat.py (hunk 3 above)
Comment (d70-t) [reply orcestra/sat.py]: Good catch!
However this leads to another issue orcestra-campaign/sattracks#1

File: orcestra/sat.py (hunk 1 above)
Comment (d70-t) [reply orcestra/sat.py:135]: Yes, indeed. In the long run, there should likely be an index which would list possible valid values. However this doesn't exist for now and thus I'd keep it with the docstring.

**Review by lkluft — APPROVED** [review]

**Pull request merged**